docs/_static/css/custom-theme.css

```diff
@@ -18,4 +80,155 @@
 
 .wy-nav-content {
     border-left: 2px #EEEEF1 solid;
-}
\ No newline at end of file
+    max-width: 1200px;
+}
+.caption {
+    border-top: 1px solid #eeeef1;
+}
+.caption-text {
+    font-family: Roboto,Arial,sans-serif;
+    font-weight:500;
+    font-size: 12px;
+    color: #666669;
+    line-height: 14px;
+    text-transform: capitalize;
+}
+
+.current {
+    font-family: Roboto,Arial,sans-serif;
+    font-weight:400;
+    font-size: 13px;
+    color: #555558;
+    line-height: 13px;
+    background: #fff;
+}
+
+.reference.internal {
+    font-family: Roboto,Arial,sans-serif;
+    font-weight:400;
+    font-size: 13px;
+    color: #555558;
+    line-height: 20px;
```

Comment:
The extra line-height here is causing the `.toctree-expand` icons to be a little off, vertically. Should update `.toctree-expand` line-height to match this value

Reply:
fixed 
<img width="66" alt="Screen Shot 2019-09-16 at 2 33 19 PM" src="https://user-images.githubusercontent.com/55099259/64983631-f3474280-d88e-11e9-8cc3-1baf93d469ce.png">